### PR TITLE
fix: variables in birth certificate

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Solves: https://github.com/opencrvs/opencrvs-core/issues/10585

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/api/certificates/source/v2.birth-certificate-certified-copy.svg [hunk 1]
```diff
@@ -378,16 +378,16 @@
...
       {{ else }}
-        {{#ifCond (lookup $declaration 'informant.relation') '===' 'FATHER'}}
+        {{#ifCond (lookup $declaration 'informant.relation') '===' 'Father'}}
           <tspan x="307" y="518.104">{{$lookup $declaration 'father.name.fullname'}}</tspan>
         {{ else }}
           <tspan x="307" y="518.104">{{$lookup $declaration 'informant.name.fullname'}}</tspan>
         {{/ifCond}}
       {{/ifCond}}
-      <tspan x="307" y="528.104">{{$intl 'v2.constants.informant' (lookup $declaration 'informant.relation')}}{{#ifCond (lookup $declaration 'informant.relation') '===' 'OTHER'}} ({{lookup $declaration 'informant.other.relation'}}) {{/ifCond}}</tspan>
+      <tspan x="307" y="528.104">{{lookup $declaration 'informant.relation'}}{{#ifCond (lookup $declaration 'informant.relation') '===' 'OTHER'}} ({{lookup $declaration 'informant.other.relation'}}) {{/ifCond}}</tspan>
```
Comment (Nil20) [inline src/api/certificates/source/v2.birth-certificate-certified-copy.svg:390]: Inspect why value of `informant.relation` could be 'Mother', 'Father' and 'OTHER' as mentioned [here](https://github.com/opencrvs/opencrvs-countryconfig/pull/1052#discussion_r2391006775)

**Review by tareq89 — APPROVED** [review]
